src/components/SearchBar.tsx

```diff
@@ -59,6 +78,19 @@ function SearchBar({ queryParams, size }: SearchBarProps) {
           onChange={handleQueryChange}
           onBlur={handleBlur}
         ></input>
+        {suggestions.length > 0 && (
+          <ul className="absolute z-50 mt-1 max-h-60 w-full overflow-auto rounded-md border border-gray-200 bg-white shadow-lg">
+            {suggestions.map((suggestion, index) => (
+              <li
+                key={index}
+                className="cursor-pointer px-4 py-2 first:rounded-t-md last:rounded-b-md hover:bg-gray-100"
```

Comment:
maybe use bg-secondary for hover here

Reply:
I put primary as bg and white text, because secondary doesn't match the search bar.